### ISSUE TITLE
yz_events:handle_info called with bad arguments [JIRA: RIAK-1567]

### ISSUE THREAD

**Issue by lukebakken** [basho/yokozuna#437]
Ticket 9603

```
CRASH REPORT Process yz_events with 0 neighbours exited with reason: no function clause matching yz_events:handle_info({#Ref<0.0.797.95953>,{chstate_v2,'riak@172.16.16.12',[{'riak@172.16.16.12',{320,63583770528}},{'riak@10.55.12.118',...},...],...}}, {state,52,<<13,127,212,199,212,213,218,124,26,252,202,146,180,233,185,0,84,193,229,150>>}) line 75 in gen_server:terminate/6 line 744
```

**Comment by ian-mi** [comment]
This appears to happen when a call to yz_cover:get_ring_used from the yz_events process times out, and the response is later left in the yz_events mailbox.

**Comment by zeeshanlakhani** [comment]
Fixed w/ #458.

**Comment by Basho-JIRA** [comment]
Ian Milligan did a great job and closed this out. 

**_[posted via JIRA by Zeeshan Lakhani]**_
